src/Reflection/ReflectionParameter.php

```diff
@@ -516,7 +533,11 @@ private function getClassName(): ?string
             return $parentClass->getName();
         }
 
-        if ($type->isBuiltin()) {
+        if ($typeHint === 'resource') {
```

Comment:
Can a `resource` type actually ever be declared? 🤔

Reply:
https://github.com/kukulich/BetterReflection/commit/a0a0b2e8bcf4e60112efe7a75f567ea0e22221d6

Probably solved by change somewhere else...